delfin/drivers/utils/ssh_client.py

```diff
@@ -0,0 +1,162 @@
+        else:
+            path = os.getcwd()
+            filename = path + '\\' + self.ssh_host + 'known_hosts'
+            file = open(filename, 'w')
+            file.write(self.ssh_private_key)
+            file.close()
+            self.ssh.load_host_keys(filename)
+
+        self.ssh.connect(hostname=self.ssh_host, port=int(self.ssh_port),
+                         username=self.ssh_username,
+                         password=self.ssh_password,
+                         timeout=self.ssh_conn_timeout)
+
+    def execCommand(self, commandStr):
+        result = None
+        try:
+            if commandStr is not None:
+                if self.ssh is not None:
+                    stdin, stdout, stderr = self.ssh.exec_command(commandStr)
+                    res, err = stdout.read(), stderr.read()
+                    re = res if res else err
+                    result = re.decode()
+                    LOG.info(
+                        'execCommand over==commandStr=={}'.format(commandStr))
+                    LOG.info('execCommand over==result=={}'.format(result))
+        except Exception as e:
+            LOG.error(e)
+            result = e
+        return result
+
+    def close(self):
+        try:
+            if self.ssh is not None:
+                # Close connection
+                self.ssh.close()
+                self.ssh = None
+                LOG.info('self.ssh Close connection success')
+        except Exception as e:
+            LOG.error(e)
+
+    def doexec(self, context, commandStr):
+        # Currently not implemented   removes command : removealert
+        """Clear alert from storage system."""
+        re = None
+        try:
+            if commandStr is not None:
+                LOG.info("sshClient commandStr==={}".format(commandStr))
+                self.connect()
+                re = self.execCommand(commandStr)
+        except paramiko.AuthenticationException as ae:
+            LOG.error('doexec==ae=={}'.format(ae))
+            raise exception.InvalidUsernameOrPassword()
+        except Exception as e:
+            LOG.error('doexec==e=={}'.format(e))
+            if 'WSAETIMEDOUT' in str(e):
+                raise exception.SSHConnectTimeout()
+            elif 'No authentication methods available' in str(e) \
+                    or 'Authentication failed' in str(e):
+                raise exception.SSHInvalidUsernameOrPassword()
+            elif 'not a valid RSA private key file' in str(e):
+                raise exception.InvalidPrivateKey()
+            elif 'not found in known_hosts' in str(e):
+                raise exception.SSHNotFoundKnownHosts(self.ssh_host)
+            else:
+                raise exception.SSHException()
+
+        finally:
+            self.close()
+        return re
+
+    def login(self, context):
+        """Test SSH connection """
+        version = ''
+        try:
+            commandStr = 'showwsapi'
```

Comment:
1. Please change all commandStr variable to command_str
2. Please remove all print statement, ex: # print(wsapiinfos)